Sources/StytchCore/StytchClient/Passkeys/StytchClient+Passkeys.swift

```diff
@@ -89,60 +88,33 @@ public extension StytchClient {
 public extension StytchClient.Passkeys {
     /// A dedicated parameters type for passkeys `register` calls.
     struct RegisterParameters: Encodable {
+        let userId: User.ID
         let domain: String
-        let username: String
+        let userAgent: String?
 
         /// - Parameters:
+        ///   - userId: The user id associated with your passkey registration.
         ///   - domain: The domain for which your passkey is to be registered.
-        ///   - username: The username associated with your user. In the future, it will be required this is a valid email address or phone number.
-        public init(domain: String, username: String) {
+        ///   - userAgent: The user agent associated with your passkey registration.
+        public init(userId: User.ID, domain: String, userAgent: String? = nil) {
+            self.userId = userId
             self.domain = domain
-            self.username = username
+            self.userAgent = userAgent
         }
     }
 
     /// A dedicated parameters type for passkeys `authenticate` calls.
-    struct AuthenticateParameters {
-        // swiftlint:disable duplicate_enum_cases
-        /// A type representing the desired request behavior
-        public enum RequestBehavior {
-            #if os(iOS)
-            /// Uses the default request behavior with a boolean flag to determine whether credentials are limited to those local on device or whether a passkey on a nearby device can be used
-            case `default`(preferLocalCredentials: Bool)
-            /// When a user selects a textfield with the `.username` textContentType, an existing local passkey will be suggested to the user.
-            case autoFill
-            #else
-            /// Uses the default request behavior
-            case `default`
-            #endif
-
-            #if os(iOS)
-            /// The RequestBehavior parameter's default value for this platform — `.default(prefersLocalCredentials: false)`
-            public static let defaultPlatformValue: RequestBehavior = .default(preferLocalCredentials: false)
-            #else
-            /// The RequestBehavior parameter's default value for this platform — `.default`
-            public static let defaultPlatformValue: RequestBehavior = .default
-            #endif
-        }
-
-        // swiftlint:enable duplicate_enum_cases
-
+    struct AuthenticateParameters: Encodable {
         let domain: String
-        let sessionDuration: Minutes
```

Comment:
I think we still want the sessionDuration for an authentication call, no? Maybe we missed that on Android (oops)